pkg/services/alerting/notifiers/slack.go

```diff
@@ -185,13 +258,21 @@ func (sn *SlackNotifier) Notify(evalContext *alerting.EvalContext) error {
 	}
 
 	body := map[string]interface{}{
+		"blocks": []map[string]interface{}{
```

Comment:
What's the reason for moving message into a block which seem to use `mrkdwn` format? Seems like notifications cannot show preview of message any longer:
![image](https://user-images.githubusercontent.com/1668778/73825307-089c1980-47fc-11ea-970b-56a60e895274.png)
![image](https://user-images.githubusercontent.com/1668778/73825593-88c27f00-47fc-11ea-92d9-de541bd6d332.png)

Reply:
Need to verify that it showed preview before

Reply:
On master I see 
![image](https://user-images.githubusercontent.com/1668778/73826016-536a6100-47fd-11ea-88a8-23b3aff1071d.png)

Reply:
@marefr It's because, based on my understanding of Slack's API specs, you need to use an attachment with Markdown in order to mention users/groups/channels. I'll read more closely though, to see if you should e.g. have a fallback message.

Reply:
@marefr I fixed the issue. The API specs are really weak on explaining this (I couldn't find the behaviour documented), but it looks as if you need to include a top-level `text` field in your message object, with just plain text (no markup) as a fallback for Web and mobile notifications. You also need the block object in order to deliver mention markup, since that doesn't work in the `text` field.